libs/ruby.json

```diff
@@ -14,5 +14,25 @@
   {"id": "rb.paypal-sdk-rest", "imports": ["paypal-sdk-rest"], "name": "paypal-sdk-rest", "repo": "paypal/PayPal-Ruby-SDK", "tags": ["payments"], "tech": ["e-commerce"]},
   {"id": "rb.mustache", "imports": ["mustache"], "name": "mustache", "repo": "mustache/mustache", "tags": ["templates"], "tech": ["ruby-web"]},
   {"id": "rb.mini-magick", "imports": ["MiniMagick"], "name": "MiniMagick", "repo": "minimagick/minimagick", "tags": ["manipulation"], "tech": ["images"]},
-  {"id": "rb.rmagick", "imports": ["RMagick"], "name": "RMagick", "repo": "rmagick-temp/rmagick", "tags": ["manipulation"], "tech": ["images"]}
+  {"id": "rb.rmagick", "imports": ["RMagick"], "name": "RMagick", "repo": "rmagick-temp/rmagick", "tags": ["manipulation"], "tech": ["images"]},
+
+  {"id": "rb.active-interaction", "imports": ["active_interaction"], "name": "ActiveInteraction", "repo": "AaronLasseigne/active_interaction", "tags": ["framework"], "tech": ["app-arch"]},
+  {"id": "rb.apotomo", "imports": ["apotomo"], "name": "Apotomo", "repo": "apotonick/apotomo", "tags": ["web-components"], "tech": ["app-arch"]},
+  {"id": "rb.cells", "imports": ["cells"], "name": "Cells", "repo": "trailblazer/cells", "tags": ["view-components"], "tech": ["app-arch"]},
+  {"id": "rb.decent-exposure", "imports": ["decent_exposure"], "name": "Decent Exposure", "repo": "hashrocket/decent_exposure", "tags": ["interfaces-for-controllers"], "tech": ["app-arch"]},
```

Comment:
maybe tags == controller-interfaces?
tech == ruby-web, because again it's rails